src/components/browse/components/SortController.js

```diff
@@ -89,36 +94,26 @@ export class SortController extends React.PureComponent {
 
         this.setState({ 'changingPage' : true }, ()=>{
             propNavigate(newHref, { 'replace' : true }, ()=>{
-                this.setState({
-                    //'sortColumn' : key,
-                    //'sortReverse' : reverse,
-                    'changingPage' : false,
-                    //'page' : 1
-                });
+                this.setState({ 'changingPage' : false });
```

Comment:
Should really rename this variable to "loading" or something.. there's no page thats' changing.